src/kaleidoscope/driver/leddriver/None.h

```diff
@@ -0,0 +1,48 @@
+/* -*- mode: c++ -*-
+ * kaleidoscope::driver::leddriver::None -- Dummy LED hardware driver for Kaleidoscope
+ * Copyright (C) 2019  Keyboard.io, Inc
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#ifndef CRGB
+
+struct cRGB {
+  uint8_t r, g, b;
+};
+
+#define CRGB(r,g,b) (cRGB){b, g, r}
+
+#endif
+
+#include "kaleidoscope/driver/leddriver/Base.h"
+
+namespace kaleidoscope {
+namespace driver {
+namespace leddriver {
```

Comment:
driver::leddriver? why not just driver::led?

Reply:
Good question. It was originally `driver::led`, but we already have stuff there that's not quite like what these things are. Having things that serve different (but kind of related) purposes within the same namespace felt wrong.

But perhaps less wrong than `driver::leddriver`...

The stuff currently in `driver::led` is something I hope to grow into a framework that can replace `cRGB`, and parts of `LEDControl`.

With that said, moving the new APIs LED drivers into `driver::led` isn't wrong. Even if there's a bit of dissonance in there, that's not going to stay that way long term.